Modules/Role/Http/Services/RoleService.php

```diff
@@ -43,32 +35,24 @@ public function submitRole(array $requestData): array
      */
     public function show(): array
     {
-        $role = Role::where('role', '!=', 'admin')
-            ->where('role', '!=', 'Content Moderator')
+        $roles = Role::where('name', '!=', Role::ROLE_SUPER)->where('name', '!=', Role::ROLE_ADMIN)->where('name', '!=', Role::ROLE_CONTENT)
             ->get();
         $allRole = [];
```

Comment:
It should be $allRoles, plural as the array contains multiple role.

Reply:
Fixed